core/org.wso2.carbon.core/src/main/java/org/wso2/carbon/core/util/CryptoUtil.java

```diff
@@ -149,14 +153,16 @@ public RegistryService getRegistryService() {
                     log.debug("Plaintext is empty. An empty array will be used as the ciphertext bytes.");
                 }
                 encryptedKey = StringUtils.EMPTY.getBytes();
-            }else{
+            } else {
                 encryptedKey = cryptoService.encrypt(plainTextBytes, algorithm, CRYPTO_API_PROVIDER_BC);
             }
 
-            if (StringUtils.isNotBlank(cipherTransformation) && returnSelfContainedCipherText) {
+            if (StringUtils.isNotBlank(cipherTransformation) && returnSelfContainedCipherText
+                    && !isSymmetricKeyEncryptionEnabled) {
 
-                Certificate certificate = cryptoService.getCertificate(CryptoContext.buildEmptyContext(
-                        MultitenantConstants.SUPER_TENANT_ID, MultitenantConstants.SUPER_TENANT_DOMAIN_NAME));
+                Certificate certificate = cryptoService.getCertificate(CryptoContext
+                        .buildEmptyContext(MultitenantConstants.SUPER_TENANT_ID,
+                                MultitenantConstants.SUPER_TENANT_DOMAIN_NAME));
```

Comment:
Same code got refactored ?

Reply:
This was due to using old wso2 check style. Fixed with 004c781ec5153daf95a6906b0c8ada5191a8c889.